tests/cnf/core/network/policy/tests/multinetpolicy_ipvlan.go

```diff
@@ -0,0 +1,589 @@
+package tests
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/openshift-kni/eco-gotests/tests/cnf/core/network/internal/netinittools"
+
+	multinetpolicyapiv1 "github.com/k8snetworkplumbingwg/multi-networkpolicy/pkg/apis/k8s.cni.cncf.io/v1beta1"
+	"github.com/openshift-kni/eco-goinfra/pkg/nad"
+	"github.com/openshift-kni/eco-goinfra/pkg/namespace"
+	"github.com/openshift-kni/eco-goinfra/pkg/networkpolicy"
+	"github.com/openshift-kni/eco-goinfra/pkg/nodes"
+	"github.com/openshift-kni/eco-goinfra/pkg/pod"
+	"github.com/openshift-kni/eco-goinfra/pkg/reportxml"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/core/network/internal/netenv"
+	"github.com/openshift-kni/eco-gotests/tests/internal/params"
+	"gopkg.in/k8snetworkplumbingwg/multus-cni.v4/pkg/types"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+var _ = Describe("Multi-NetworkPolicy : IPVLAN CNI", Ordered, Label("ipvlancni"), ContinueOnFailure, func() {
+
+	var (
+		sriovInterfacesUnderTest                         []string
+		tNs1, tNs2                                       *namespace.Builder
+		testPod1, testPod2, testPod3, testPod4, testPod5 *pod.Builder
+		ports                                            = []string{"5001", "5002", "5003"}
+		protocols                                        = []string{"tcp", "tcp", "udp"}
+		allOpen                                          = map[string]string{"5001": "pass", "5002": "pass", "5003": "pass"}
+		allClose                                         = map[string]string{"5001": "fail", "5002": "fail", "5003": "fail"}
+		p5001Open                                        = map[string]string{"5001": "pass", "5002": "fail", "5003": "fail"}
+		p5001p5002Open                                   = map[string]string{"5001": "pass", "5002": "pass", "5003": "fail"}
+	)
+
+	const (
+		testNs1, testNs2             = "policy-ns1", "policy-ns2"
+		pod1, pod2, pod3, pod4, pod5 = "pod1", "pod2", "pod3", "pod4", "pod5"
+	)
+
+	testData := podsData{
+		"pod1": {IPv4: "192.168.10.10/24", IPv6: "2001:0:0:1::10/64", Protocols: protocols, Ports: ports},
+		"pod2": {IPv4: "192.168.10.11/24", IPv6: "2001:0:0:1::11/64", Protocols: protocols, Ports: ports},
+		"pod3": {IPv4: "192.168.10.12/24", IPv6: "2001:0:0:1::12/64", Protocols: protocols, Ports: ports},
+		"pod4": {IPv4: "192.168.20.11/24", IPv6: "2001:0:0:2::11/64", Protocols: protocols, Ports: ports},
+		"pod5": {IPv4: "192.168.20.12/24", IPv6: "2001:0:0:2::12/64", Protocols: protocols, Ports: ports},
+	}
```

Comment:
It looks like you used very similar var and const in the bond  tests. If so, could you move them to the internal folder?